fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1848,6 +1850,53 @@ Future<Optional<std::string>> AdvanceVersionImpl::commit(ReadYourWritesTransacti
 	return Optional<std::string>();
 }
 
+ACTOR static Future<RangeResult> getVersionEpochActor(ReadYourWritesTransaction* ryw, KeyRangeRef kr) {
+	ryw->getTransaction().setOption(FDBTransactionOptions::LOCK_AWARE);
+	ryw->getTransaction().setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
```

Comment:
It doesn't actually change the behavior in this case, but I've just gone through all of the transactions in this file and changed `READ_SYSTEM_KEYS` and `ACCESS_SYSTEM_KEYS` to `RAW_ACCESS`. The native transaction doesn't require setting system key access, and the choice above is meant to avoid accidentally setting these options on the RYW transaction. If we did, then the system key access would leak to the caller.